src/libs/suggest.js

```diff
@@ -24,6 +25,15 @@ export const selectItem = (selectedItem, replaceUrl = false) => {
     window.app.navigateTo(`/places/?type=${selectedItem.name}`,
       {}, { replace: replaceUrl });
   } else if (selectedItem instanceof Intention) {
+    Telemetry.add(
+      Telemetry.SUGGEST_CLICK,
+      null,
+      null,
+      {
+        useNlu: geocoderConfig.useNlu,
+        isCategory: !!selectedItem.category,
```

Comment:
@amatissart proposes to send the category itself, instead of just its presence, to allow for more analysis. I agree this would be useful.

Reply:
Yes the name of the category would be useful.  
And maybe some more details about the presence of a query or a place, such as 
```
hasFullTextQuery: !!selectedItem.fullTextQuery,
hasPlace: !!selectedItem.place
```